src/Core/Types/Types/Relay/IdFieldValueSerializerFactory.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+#nullable enable
+
+namespace HotChocolate.Types.Relay
+{
+    public class IdFieldValueSerializerFactory : IIdFieldValueSerializerFactory
```

Comment:
Why introducing a new interface on V10?

Reply:
Sorry for the confusion, in Slack I'd said:
> If all g, I'll make the same update for v11

Would you be ok with this approach if I make the same type of change for v11?